src/stan/math/matrix/append_row.hpp

```diff
@@ -46,7 +46,7 @@ namespace stan {
     template <typename T1, typename T2, int R1, int R2>
     inline Matrix<typename return_type<T1, T2>::type, Dynamic, 1>
     append_row(const Matrix<T1, R1, 1> & A,
-          const Matrix<T2, R1, 1> & B) {          
+               const Matrix<T2, R1, 1> & B) {          
```

Comment:
That signature will never get called because
there's no way to infer R2.  The whole thing should
have been removed.  I'll do that and push.

- Bob

> On Jan 26, 2015, at 4:16 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/math/matrix/append_row.hpp:
> 
> > @@ -46,7 +46,7 @@ namespace stan {
> >      template <typename T1, typename T2, int R1, int R2>
> >      inline Matrix<typename return_type<T1, T2>::type, Dynamic, 1>
> >      append_row(const Matrix<T1, R1, 1> & A,
> > -          const Matrix<T2, R1, 1> & B) {          
> > +               const Matrix<T2, R1, 1> & B) {          
> 
> @bob-carpenter, should this line have R2 instead of R1?
> 
> —
> Reply to this email directly or view it on GitHub.
>